be/src/common/config.h

```diff
@@ -874,6 +874,12 @@ CONF_Int32(pipeline_executor_size, "0");
 // Will remove after fully test.
 CONF_Bool(enable_index_apply_preds_except_leafnode_of_andnode, "false");
 
+// If the memory consumption of sort node exceed this limit, will trigger spill to disk;
+// Set to 0 to disable; defaul to 1G; min: 128M
+CONF_mInt64(external_sort_bytes_threshold, "1073741824");
```

Comment:
Not in be.conf， pass it by using FE's session variable.

Reply:
The default value is 0 or -1, it means spill is disabled. If user want to enable it, it should be a value larger than 0.

Reply:
Please also add the config to fe fuzzy, we could choose the fuzzy value in (0(disabled), 1 (almost always enabled), 1MB, 100gb))